jenkins-build-pack-api/Jenkinsfile

```diff
@@ -727,6 +809,49 @@ def loadBuildModules(buildModuleUrl){
 		environmentDeploymentMetadata = load "environment-deployment-metadata-loader.groovy"
 		echo "Environment deployment data loader module loaded successfully."
 
+		if (configLoader.APIGEE.ENABLED) {
```

Comment:
Do we need something like this - if (configLoader.APIGEE && configLoader.APIGEE.ENABLED) {.... ?

Reply:
updated